mailet/standard/src/test/java/org/apache/james/transport/matchers/HasMimeTypeParameterTest.java

```diff
@@ -0,0 +1,119 @@
+package org.apache.james.transport.matchers;
```

Comment:
Missing license :trollface:

Reply:
if you need a proof that intellij doesn't like me